spec/spec_helper.rb

```diff
@@ -0,0 +1,41 @@
+
+require 'rubygems'
+
+# Configure Rails Environment
+ENV["RAILS_ENV"] ||= 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.